lib/iris/fileformats/grib/message.py

```diff
@@ -124,6 +160,15 @@ def data(self):
             raise TranslationError(fmt.format(template))
         return data
 
+    def __getstate__(self):
```

Comment:
Okay, on reflection I think you're right -- previous behaviour has not changed anyway, and this tweak just sidesteps the resulting exception.